sty/presentation/presentation.dtx

```diff
@@ -625,6 +626,7 @@
 % greater than the current precedence specified by |\pres@current@precedence|
 %    \begin{macrocode}
 \def\PrecWrite#1{\ifnum\pres@p>\pres@current@precedence\else{#1}\fi}
+\def\PrepostPrecWrite#1{\ifnum\pres@p@key>\pres@infty@minusone\else{#1}\fi}

```

Comment:
why does prepost need their own precedences? I am not sure I understand. Please comment.